core/src/Revolution/Processors/Workspace/Lexicon/ReloadFromBase.php

```diff
@@ -41,42 +41,55 @@ public function getLanguageTopics()
      */
     public function process()
     {
-        $language = 'en';
-        $namespace = 'core';
-        $corePath = $this->modx->getOption('core_path', null, MODX_CORE_PATH);
+        $namespace = htmlentities($_POST['namespace']);
+        $language = htmlentities($_POST['language']);
+        $topic = htmlentities($_POST['topicLexicon']);
+        if ($namespace == 'core') {
+            $namespacePath = $this->modx->getOption('core_path', null, MODX_CORE_PATH);
+        } else {
+            $namespacePath = $this->modx->getOption('core_path', null, MODX_CORE_PATH) . 'components/' . $namespace;
```

Comment:
The path to a namespace should not be hardcode like this since it can be in a different path as well.

You have to ask this info from the right namespace object. See: https://github.com/modxcms/revolution/blob/1e7c048833a8419c113a0f10c0daa3bcad5b4d8e/core/src/Revolution/modLexicon.php#L363-L382